test-blink.cpp

```diff
@@ -32,19 +35,49 @@ int main(void)
         return 1;
     }
 
-    wiringPiSetupGpio();
-    pinMode(gpio, OUTPUT);
+    // Setup GPIO with libgpiod
+    struct gpiod_chip *chip;
+
+    chip = gpiod_chip_open("/dev/gpiochip0");
+    if (!chip)
+    {
+        std::cerr << "Failed to open gpiochip: /dev/gpiochip0" << std::endl;
+        return 1;
+    }
+
+    line = gpiod_chip_get_line(chip, gpio);
+    if (!line)
+    {
+        std::cerr << "Failed to get line! Output code: " << envGPIO << std::endl;
+        return 1;
+    }
+
+    int ret = gpiod_line_request_output(line, CONSUMER, 0);
+    if (ret < 0)
+    {
+        std::cerr << "Request line as output failed! Output code: " << ret << std::endl;
```

Comment:
```suggestion
        std::cerr << "Failed to set gpio line as output." << std::endl;
```
The output code is only useful for error handling. It will always be -1 on error.